Dockerfile

```
@@ -0,0 +1,13 @@
+FROM oracle/graalvm-ce:19.3.1-java8 AS BUILD
+RUN curl -o - -s http://www.nic.funet.fi/pub/mirrors/apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz | tar xz -C /usr/local
+RUN gu install native-image
+COPY . /build
+WORKDIR /build
+RUN /usr/local/apache-maven-3.6.3/bin/mvn
+
+FROM ubuntu:20.04
+RUN apt-get -qq update && apt-get -y -qq install libxi6 libxtst6 libxrender1
```

1. These packages are not enough.
2. The apt-get generate garbage during its work. Therefore that garbage should be cleared.

I think, this line should be rewritten like this:

```suggestion
RUN apt-get -qq update \
 && apt-get -y -qq install libxi6 libxtst6 libfreetype6 libpng16-16 libfontconfig1 libjpeg8 fonts-dejavu-extra graphviz \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

---

BTW, thanks for introducing the Suggested change functionality to me, I hadn't seen it before.